Common/Util/Composer.cs

```diff
@@ -132,7 +132,7 @@ public Composer()
 
         private CompositionContainer _compositionContainer;
         private readonly List<Type> _exportedTypes = new List<Type>();
-        private readonly Task<List<ComposablePartDefinition>> _composableParts;
+        private readonly Task<IEnumerable<ComposablePartDefinition>> _composableParts;
```

Comment:
Interesting, returning a list in `master` has the benefit that it's performed in _parallel_ in the task were returning an IEnumerable will deffer the operation to **each** request _synchronously_ which will delay requests to the composer

Reply:
That's certainly not intended. I'm reverting this.